mora/service/common.py

```diff
@@ -115,11 +116,14 @@ def get_obj_value(obj, path: tuple, filter_fn: Callable = None):
         return props
 
 
-def ensure_bounds(valid_from: str,
-                  valid_to: str,
+def ensure_bounds(valid_from: datetime.datetime,
+                  valid_to: datetime.datetime,
                   props: List[FieldTuple],
                   obj: dict,
                   payload: dict):
+    def filter_fn(prop):
+        return util.from_iso_time(prop['virkning']['from'])
```

Comment:
We should factor this out to a separate function, such as `get_effect_from()` in `common.py` or `util.py`.